sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/OnTimerMethodSpecifier.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.beam.sdk.transforms.reflect;
```

Comment:
The RAT check (**R**elease **A**udit **T**ool) fails the build fast because you need the Apache License header here.